src/azure-cli-core/azure/cli/core/auth/credential_adaptor.py

```diff
@@ -57,12 +57,13 @@ def signed_session(self, session=None):
     def get_token(self, *scopes, **kwargs):
         logger.debug("CredentialAdaptor.get_token: scopes=%r, kwargs=%r", scopes, kwargs)
 
-        # SDK azure-keyvault-keys 4.5.0b5 passes tenant_id as kwargs, but we don't support tenant_id for now,
-        # so discard it.
-        kwargs.pop('tenant_id', None)
+        # Discard unsupported kwargs: tenant_id, enable_cae
+        filtered_kwargs = {}
+        if 'data' in kwargs:
+            filtered_kwargs['data'] = kwargs['data']
```

Comment:
`claims` is a supported keyword argument of `UserCredential.get_token`:

https://github.com/Azure/azure-cli/blob/95ef12647b830f758b73db547e8166ef9a673113/src/azure-cli-core/azure/cli/core/auth/msal_credentials.py#L55

Not passing `claims` breaks CAE silent re-authentication. Interestingly, no issue is reported. 🤔